front/components/assistant/conversation/input_bar/InputBar.tsx

```diff
@@ -361,6 +361,7 @@ export function FixedAssistantInputBar({
   stickyMentions,
   conversationId,
   additionalAgentConfiguration,
+  // TODO:
   actions = INPUT_BAR_ACTIONS.slice(),
```

Comment:
@philipperolet any idea of what you were trying to do here?

Reply:
I meant why the `.slice`?

Reply:
Oh! Since INPUT_BAR_ACTIONS is readonly, can't be assigned to `actions` which is not

Reply:
Fixed then, we should never use `.slice()` in default params.